.github/CONTRIBUTING.md

```diff
@@ -1,159 +0,0 @@
-# How to contribute
-
-Welcome to our ever growing community! We are more than happy to accept external contributions to the project in the form of feedback, [translations](http://flyve.org/glpi-plugin/howtos/contribute-translating), bug reports and even better, pull requests!
```

Comment:
Hi, @bootbeep 
What's about the relative links?

Reply:
They're broken, 404, I'll try another thing... Any suggestion?

Reply:
:goberserk:

Reply:
Ok, good luck.

I will try to help you, if does not works, I prefer remove specific links and use generic URL like the Flyve MDM org on Transifex or GitHub.

Reply:
> They're broken, 404, I'll try another thing... Any suggestion?

I was able to open the URLs without a 404. Please use the full URL because sometimes this docs are open on local computers and it's more user friendly to have them on that way.